apps/solhint/src/app/SolhintPluginClient.ts

```diff
@@ -0,0 +1,145 @@
+import { PluginClient } from '@remixproject/plugin'
+import { createClient } from '@remixproject/plugin-webview'
+import EventEmitter from 'events'
+import { processStr } from 'solhint'
+import { applyExtends } from 'solhint/lib/config/config-file'
```

Comment:
isn't everything properly exported?

Reply:
they are. Had a resolution issue but it appears to be non existent.